### ISSUE TITLE
Message on error

### ISSUE THREAD

**Issue by htfy96** [websanova/vue-auth#190]
Currently `error: function() {}` option in methods does not receive any information regarding the type and details of error. 
In my case, the server returns something like `{ "msg": "username does not exist" }` or `{ "msg" : "Bad password" }` and I want to display `msg` on the page.
 Is there any way to implement this?

**Comment by heisjoe** [comment]
@htfy96  
in err callback function ,you should use err.response.data to show your err message

**Comment by websanova** [comment]
Not sure what you mean, like as global error, or just during login, register calls, etc?

**Comment by htfy96** [comment]
@heisjoe That's exactly what I need 👍 , though it is undocumented in `register` and `login` documentation.

**Comment by websanova** [comment]
Well, that params are passed directly to Vue resource or axios so it's not
really anything to do with the plugin

**Comment by htfy96** [comment]
I found that invoking `refresh` with no Internet access would yield a uncaught promise from axios without notifying the `error()` function:
```javascript
  _invalidToken: function (res) {
>   if (res.status === 401) { // Cannot read property 'status' of undefined.
      this.options.logoutProcess.call(this, res, {redirect: this.options.authRedirect});
    }
  },
---
    function _responseIntercept(res) {
        var token;

        if (this.options.http._invalidToken) {
>           this.options.http._invalidToken.call(this, res); // res === undefined
        }
---
  if (res) {
      this.options.Vue.axios.interceptors.response.use(function (response) {
        res.call(_this, response);
        return response;
      }, function (error) {
>       res.call(_this, error.response); // error = Error: Network Error at createError. error.response === undefined
        return Promise.reject(error);
      })
```

It seems that Axios driver access `.status` at any case yielding the above error. Is there any way to handle network error like this?

**Comment by websanova** [comment]
Ok, this "should" be fixed in `v2.13.2-beta`. I've just added a check for the `error.response` to exists before calling it in the  auth.

Actually in one of my projects I have a check for this, but I use vue-resource so it just passes the `response` in directly. But it's the same idea, I have an interceptor like this for the app.

```
Vue.http.interceptors.push(function (req, next) {
    next(function (res) {
        if ( ! res.ok) {
            _this.$router.push({name: 'error-502'})
        }
    });
});
```
